megatron/model/rwkv/v6/rwkv.py

```diff
@@ -104,7 +126,7 @@ class RWKV_TimeMix(nn.Module):
     TODO: fix jit compiling.
```

Comment:
Is this based on the parser issue we discussed? I think it's worth testing just-jit and reordered jit and heuristics like I suggested before merging with this TODO